odoo/model/invoice_category.go

```diff
@@ -0,0 +1,82 @@
+package model
+
```

Comment:
I thought I put the currently known Odoo models into their own package.
Is that ok or do you prefer a flatter hierarchy where the models are in the same package as the client?

Reply:
I think it's a good idea to keep the "specific" models in its own package. I like it.

Reply:
I probably would have kept a flatter hierarchy. But this very much personal preference and I'm also happy with putting it into its own package.